guides/common/modules/proc_registering-a-host-using-the-bootstrap-script.adoc

```diff
@@ -4,7 +4,7 @@
 **Deprecated** Use registration via Global registration feature.
 
 Use the bootstrap script to automate content registration and Puppet configuration.
-You can use the bootstrap script to register new hosts, or to migrate existing hosts to {ProjectNameX} from {Project} 5, RHN, SAM, or RHSM.
+You can use the bootstrap script to register new hosts, or to migrate existing hosts from RHN, SAM, RHSM, or an earlier version of {ProjectName}.
```

Comment:
@ekohl How about this?

Reply:
Technically I think it can be used on hosts registered to any other Satellite/Foreman/Katello instance. So it can also be used to migrate hosts from a Satellite 7 to another Satellite 7, for example if you had built a lab environment but then another lab/production. Perhaps `or a different {ProjectName} (instance)` would be a more generic description? Not sure about the exact phrasing, but I hope you get the idea.

Reply:
Cool. I hope this will suffice.